core-kit/next/sys-kernel/debian-sources/debian-sources-6.9.12_p1.ebuild

```diff
@@ -317,15 +317,17 @@ src_install() {
 			--kerneldir=${S} \
 			--bootdir=${D}/boot \
 			--cachedir=${WORKDIR}/genkernel-cache \
+			--no-clear-cachedir \
 			--kernel-modules-prefix=${D} \
+			--ramdisk-modules \
 			--initramfs-filename=initramfs-${KERN_SUFFIX} || \
 				die "genkernel failed:  $?" \
 	)
 
 	# copy the fresh Genkernel cache into the image
 	if use genkernel; then
 		dodir /var/cache/genkernel
-		cp "${WORKDIR}/genkernel-cache/*" "${D}/var/cache/genkernel/" || die
+		cp -r "${WORKDIR}/genkernel-cache/4.3.10" "${D}/var/cache/genkernel/" || die
```

Comment:
genkernel is autogened now. If we get a new version of genkernel e.g. 4,3.11, could this break again?

Reply:
It is pinned in the autogen YAML.  It definitely will break again if someone bumps the version without testing it --- this version was the last one without further complications. We are unlikely ever to get another version --- after this I'll do `dracut` and likely make that default because it is well-supported and reliable, and Macaroni is using it too.